packages/mad-tag-ocr/package.json

```diff
@@ -0,0 +1,55 @@
+{
+    "name": "@equinor/react-native-mad-tag-ocr",
+    "version": "1.0.0",
```

Comment:
Do you feel comfortable majoring this to 1 on first release? If you suspect there will potentially be breaking changes to the package going forward, maybe majoring it to 0 to begin with is better.